components/progress-ring/private/ring-shape.jsx

```diff
@@ -61,7 +61,7 @@ const calculateD = (fillPercent) => {
  * Displays the progress ring shape.
  */
 const ProgressRingShape = (props) => {
-	const progressStyles = {};
+	const progressStyles = { height: '2rem' };
```

Comment:
This does not work correctly for `medium` sized progress rings. Please view the "Snapshot Test Examples" story to see what I'm talking about. To fix this, the `size` prop needs to be passed to the `ProgressRingShape` component in order to determine this value... if it is `'large'`, use `2rem`, otherwise use `1.5rem`:
```
const progressStyles = { height: (props.size === 'large') ? '2rem' : '1.5rem' };
```

<img width="1150" alt="Screen Shot 2019-07-09 at 4 33 46 PM" src="https://user-images.githubusercontent.com/991513/60927580-ab40e980-a267-11e9-80c5-f9b724feed19.png">

Reply:
My branch got corrupted and I created a new PR with the changes you requested. Please take a look: 
https://github.com/salesforce/design-system-react/pull/2155